checker/tests/nullness/java8inference/Issue980.java

```diff
@@ -24,6 +24,8 @@ void m(List<String> strings) {
         // This assignment issues a warning due to incompatible types:
         List<String> collectedStrings = s.collect(Collectors.toList());
 
+        // This issues an error because type argument inference fails on the above statement.
```

Comment:
This error is a pity, because this is a false positive caused by the type argument inference failure.
On the statement above, can we use a more optimistic type (unless we provide conservative flags)?